marlowe-dashboard-client/src/AppM.purs

```diff
@@ -1,19 +1,24 @@
-module AppM where
+module AppM
+  ( AppM
+  , runAppM
+  ) where
 
 import Prologue
 import Clipboard (class MonadClipboard, copy)
-import Control.Monad.Reader.Trans (class MonadAsk, ReaderT, asks, runReaderT)
+import Control.Monad.Reader.Class (class MonadAsk)
 import Effect.Aff (Aff)
 import Effect.Aff.Class (class MonadAff)
 import Effect.Class (class MonadEffect, liftEffect)
-import Env (Env)
-import Type.Equality (class TypeEquals, from)
+import Halogen (Component)
+import Halogen.Store.Monad (class MonadStore, StoreT, getStore, runStoreT)
+import Safe.Coerce (coerce)
+import Store as Store
 
 newtype AppM a
-  = AppM (ReaderT Env Aff a)
+  = AppM (StoreT Store.Action Store.Store Aff a)
```

Comment:
I agree with @paluh, I don't think it is a good idea to merge `Env` and `Store`. `Store` should be global mutable state, and `Env` should be global readonly state. I'd prefer it if this was 

```suggestion
  = AppM (ReaderT Env (StoreT Store.Action Store.Store Aff) a)
```

Reply:
Ah, I see there aren't instances for `MonadStore` `ReaderT`. Hence, `ReaderT Env (StoreT S m) a` won't have a `MonadStore` instance... What the heck is the matter with this library?! This is not at all the way monad transformers are supposed to work... 😞